include/crm/common/roles_internal.h

```diff
@@ -57,7 +57,7 @@ extern "C" {
  *
  * \return Node that \p rsc is active on, if any, otherwise NULL
  */
-static inline pe_node_t *
```

Comment:
copyright on this file; maybe others, run a check

Also surprised this worked before, since `pe_node_t` in in a compat header

Reply:
Bah, that was a rebase error